app/lib/dartdoc/dartdoc_runner.dart

```diff
@@ -15,6 +15,7 @@ import '../job/backend.dart';
 import '../job/job.dart';
 import '../scorecard/backend.dart';
 import '../scorecard/models.dart';
+import '../search/text_utils.dart' show isSelfDocumenting;
```

Comment:
Maybe we should move this file to `shared/` or split out the non-search specific parts and put them in `shared/`..

I know it's hard to get perfect, but if we could strive to avoid too many cross-dependencies that might be nice :)